EdenGri/sonic-utilities
# Update the DB Migrator to remove the unused jinja2 cache in case of a warm upgrade

#### What I did
Update the DB Migrator to remove the unused jinja2 cache in case of a warm upgrade

#### How I did it
Add a new version to the db_migrator that will delete the JINJA2 cache from the LOGLEVEL DB

#### How to verify it
1. start with a switch that has an old image (old image means that the VERSION in the config db is 3_0_5 or lower)
2. install this image to the switch
3. run warm-reboot
4. verify that the LOGLEVEL DB does not contain jinja2 cache -> run redis-cli -n 3 keys "*JIN*" (expect to see empty array)

#### Previous command output (if the output of a command-line utility has changed)

#### New command output (if the output of a command-line utility has changed)
